pkg/execute/executor.go

```diff
@@ -234,19 +235,34 @@ func runPingCommand(args []string, clusterName string) string {
 	return fmt.Sprintf("pong from cluster '%s'", clusterName)
 }
 
-func showControllerConfig() (string, error) {
+func showControllerConfig() (configYaml string, err error) {
 	configPath := os.Getenv("CONFIG_PATH")
 	configFile := filepath.Join(configPath, config.ConfigFileName)
 	file, err := os.Open(configFile)
 	defer file.Close()
 	if err != nil {
-		return "", err
+		return configYaml, err
 	}
 
 	b, err := ioutil.ReadAll(file)
 	if err != nil {
-		return "", err
+		return configYaml, err
 	}
 
-	return string(b), nil
+	c := &config.Config{}
+	if len(b) != 0 {
+		yaml.Unmarshal(b, c)
```

Comment:
do we expect any errors in unmarshal ?

Reply:
In the worst case, we will get empty values of keys don't match. Will handle error though